scripts/pylib/twister/twisterlib/harness.py

```diff
@@ -158,6 +159,13 @@ def handle(self, line):
         else:
             tc.status = "failed"
 
+class ShellCommand:
```

Comment:
I'm not a particular owner of this file, but I'd be hesitant to add a class here for a single example that only shows string formatting.

Reply:
I'm not seeing the benefit of making a class from this, can be done inline on line 218